docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -567,19 +498,84 @@ Write operations are supported for tables stored on the following systems:
   make sure that no concurrent data modifications are run to avoid data
   corruption.
 
+(delta-lake-schema-table-management)=
+
 ### Schema and table management
 
 The {ref}`sql-schema-table-management` functionality includes support for:
 
-- {doc}`/sql/create-schema`
-- {doc}`/sql/drop-schema`
-- {doc}`/sql/alter-schema`
-- {doc}`/sql/create-table`
+- {doc}`/sql/create-table`, see also {ref}`delta-lake-sql-basic-usage`
 - {doc}`/sql/create-table-as`
 - {doc}`/sql/drop-table`
-- {doc}`/sql/alter-table`
+- {doc}`/sql/alter-table` , see also {ref}`delta-lake-alter-table`
```

Comment:
there are two section .. alter table execute and alter table rename .. maybe consolidate as one with subesctions